readme.md

```diff
@@ -3,15 +3,15 @@
 # Skipper
 
 Skipper is an HTTP router that acts as a reverse proxy (with support for flexible route definitions), alters
-requests, and responses with filters. It:
+requests, and responds with filters. It:
```

Comment:
what I meant here is that both the request and the response can be altered 'by' the filters. this sounds more as the filters were the response.